File: code-signal-arcade-universe/intro/python3/lucky_ticket.py
def is_lucky(n):
    """
    Number -> Boolean, given a number with even number of digits, returns
    true if sum of first half of digits is equal to sum of second half
    """
    lst = [int(x) for x in list(str(n))]
    half = len(lst)//2
    fh = lst[:half]
    sh = lst[half:]
    return sum(fh) == sum(sh)

File: code-signal-arcade-universe/intro/clojure/lucky_ticket.clj
(defn is-lucky
  "Number -> Boolean, given a number with even number of digits, returns true if sum of first half of digits is equal to sum of second half"
  [n]
  (let [lon (map #(Integer/parseInt %) (clojure.string/split (str n) #""))
        spl (split-at (/ (count lon) 2) lon)
        sms (map #(reduce + %) spl)]
    (apply = sms)))
